src/renderer/i18n/locales/en-US.js

```diff
@@ -991,6 +991,7 @@ export default {
       HTLC_CLAIM: 'Timelock Claim could not be created',
       HTLC_REFUND: 'Timelock Refund could not be created',
       MULTI_PAYMENT: 'Multi payment could not be created',
+      MULTI_PAYMENT_TO_SELF: 'Excluding {amount} the sender sent to themselves',
```

Comment:
FYI we eventually settled on `themself` instead of `themselves` on the explorer (https://dictionary.cambridge.org/dictionary/english/themself).

Reply:
I'm going with `Excluding {amount} sent to itself` since I'm not happy with either to be honest.

Reply:
Agree that neither `themself` nor `themselves` sounds natural in this context, ditto with the explorer.